client/config/defaults.js

```diff
@@ -7,6 +7,10 @@ import { AutoForm } from "meteor/aldeed:autoform";
  * A place to put misc. package configurations
  */
 
+// Set the default React app container inside which the rest of the UI will live
+// This enables the ability to completely override the look and feel of RC
+export const DEFAULT_APP_CONTAINER = "CoreReactionApp";
```

Comment:
What's the need to set a session variable for the default app container?

Could you not just override it by using `replaceComponent("CoreReactionApp", MyCustomApp);`

Reply:
I don't think I have a good response here. I'd gotten used to seeing registerComponent in digging through the code. Going the replaceComponent route simplifies things a bit so I'm onboard.